crossing-training.lic

```diff
@@ -738,11 +738,16 @@ class CrossingTraining
   end
 
   def train_stealth
-    hide_in 851
-    hide_in 850
-    hide_in 764
-    hide_in 5992
-    hide_in 992
+    if DRSkill.getrank('Stealth') >= 30
+      echo('Skill too high to learn this way, removing Stealth from training')
+      @settings.crossing_training.delete('Stealth')
```

Comment:
I prefer something like
```
if ...
    ...
    return
end

hide_in ...
```

It's a completely stylistic thing, so don't worry about changing it, I just wanted to point out an alternative.

Reply:
that's what I had originally, I'll use that style in the future.